bool use_guided
float64 push_dist
---
Header header
geometry_msgs/Point start_point
float64 push_angle
uint32 num_objects
